lib/YaST/workarounds.pm

```diff
@@ -48,4 +49,26 @@ sub apply_workaround_bsc1204176 {
     }
 }
 
+=head2 apply_workaround_bsc1207157 ():
+
+Workaround for the iscsi return code issue.
+
+Records a soft failure with a reference to bsc#1207157
+
+Changes the iscsid service file to require and start after the iscsid socket.
+Then reloads systemd, in order to scan for the changed unit.
+
+=cut
+
+sub apply_workaround_bsc1207157 {
+    record_soft_failure('bsc#1207157 - openQA test fails in iscsi_client - launched then no response');
+    my $service_unit = '/usr/lib/systemd/system/iscsid.service';
+    # append the two lines at the end of the [Unit] section of the service file, as specified in bsc#1206132
+    my $cmd = q(awk -i inplace 'BEGIN {c=1}; /^$/ {if (c){print("Requires=iscsid.socket\nAfter=iscsid.socket")} c=0}; {print}' );
```

Comment:
please, use [Config::Tiny](https://metacpan.org/pod/Config::Tiny) to modify the INI file, it will be more maintainable as we don't know when this will be fixed, this bug might stay for us for a while.